src/config.rs

```diff
@@ -800,6 +810,7 @@ impl<'a> Cfg<'a> {
             false,
             self,
         )?;
+        options.skip_std = skip_std;
```

Comment:
I'm sorry, but this feels very strange. Why are other args passed to  the constructor, but this one updated from the outside?

@djc What do you think?

Reply:
I think I basically suggested this in https://github.com/rust-lang/rustup/pull/4587#discussion_r2491261138? It seems nicer to me to not bother all callers with supplying a default value.

Reply:
@djc Oh I'm sorry for commenting on the suggested revision, but due to an web-related problem (website crash) it seems that I cannot properly see what that comment was trying to address 🙇 Closing this thread as resolved now.